lib/cib/cib_utils.c

```diff
@@ -465,18 +465,17 @@ cib_perform_op(const char *op, int call_options, cib_op_t * fn, gboolean is_quer
 }
 
 xmlNode *
-cib_create_op(int call_id, const char *token, const char *op, const char *host, const char *section,
-              xmlNode * data, int call_options, const char *user_name)
+cib_create_op(int call_id, const char *op, const char *host,
+              const char *section, xmlNode *data, int call_options,
+              const char *user_name)
 {
     xmlNode *op_msg = create_xml_node(NULL, "cib_command");
 
     CRM_CHECK(op_msg != NULL, return NULL);
-    CRM_CHECK(token != NULL, return NULL);
 
     crm_xml_add(op_msg, F_XML_TAGNAME, "cib_command");
 
     crm_xml_add(op_msg, F_TYPE, T_CIB);
-    crm_xml_add(op_msg, F_CIB_CALLBACK_TOKEN, token);
```

Comment:
Also test a mixed-version cluster. The DC sets the pacemaker-based on its node as "the writer", and I believe all write requests are forwarded to it. Not sure if this is passed unchanged, but if the writer is older, it will still insist that messages have a token.

Reply:
> test running command-line CIB clients from both a Pacemaker Remote node and a CIB_remote host to be sure.

Sure, I can test that in a bit.

> Also test a mixed-version cluster. The DC sets the pacemaker-based on its node as "the writer", and I believe all write requests are forwarded to it. Not sure if this is passed unchanged, but if the writer is older, it will still insist that messages have a token.

I don't see how this presence or absence of the token makes a difference. A DC that doesn't expect a token will ignore it, and a DC that expects a token to grab a token will get a NULL value that it doesn't use anyway. How would the missing token change behavior?

Reply:
Also AFAICT the tokens are added to a message only via `cib_create_op()`, which is called only by `cib_t` variant methods. One node won't be aware of another node's `cib_t` object, so we don't have to worry about the `CRM_CHECK()` line in `cib_create_op()` if that was the concern

Reply:
I haven't looked at how requests get forwarded to the writer. If they're simply passed along as-is, then the newer client will not add a token, but the older writer will still have the CRM_CHECK() and reject it.

Reply:
(if the same code winds up processing the message -- I haven't checked)

Reply:
Great. Let me know when you're ready to merge